components/svg/DayIcon.js

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import SvgUri from "react-native-svg-uri";
+export default () => (
+  <SvgUri
```

Comment:
React-native-svg is more reliable in my opinion. Last update for react-native-svg-uri was 2 years ago.

Reply:
@gulagha as far as I know, react-native-svg-uri is a wrapper around react-native-svg. Please have a look: https://github.com/vault-development/react-native-svg-uri

Reply:
@jmuseyibli that's right, but the react-native-svg version was 4.3.1 at that time, now it's 12.1.1

Reply:
Okay. I see your point. Gulagha, if you have time, please checkout this branch, and change the library.

Reply:
Yep, sure

Reply:
Thanks!! :)